tests/rados/test_osd_scrub_load_threshold_parameter.py

```diff
@@ -0,0 +1,548 @@
+    rados_object = RadosOrchestrator(node=cephadm)
+    scrub_object = RadosScrubber(node=cephadm)
+    mon_obj = MonConfigMethods(rados_obj=rados_object)
+    installer = ceph_cluster.get_nodes(role="installer")[0]
+    wait_time = 120
+    osd_nodes = ceph_cluster.get_nodes(role="osd")
+    replicated_config = config.get("replicated_pool")
+    pool_name = replicated_config["pool_name"]
+
+    try:
+
+        log_line_no_scrub = "'scrub_load_below_threshold:.* = no'"
+        log_line_scrub = "'scrub_load_below_threshold:.* = yes'"
+        # enable the file logging
+        if not rados_object.enable_file_logging():
+            log.error("Error while setting config to enable logging into file")
+            return 1
+        log.info("Logging to file configured")
+        # disable autoscaler
+        assert mon_obj.set_config(
+            section="global", name="osd_pool_default_pg_autoscale_mode", value="off"
+        ), "Could not set pg_autoscale mode to OFF"
+
+        if not rados_object.create_pool(**replicated_config):
+            log.error("Failed to create the replicated Pool")
+            return 1
+
+        rados_object.bench_write(pool_name=pool_name, byte_size="5K", max_objs=10000)
+        msg_data_push = f"The data pushed into the {pool_name} pool"
+        log.info(msg_data_push)
+
+        # Get the pool pg_id
+        pg_id = rados_object.get_pgid(pool_name=pool_name)
+        pg_id = pg_id[0]
+        msg_pool_id = f"The {pool_name} pg id is - {pg_id}"
+        log.info(msg_pool_id)
+
+        # Get the acting PG set
+        acting_pg_set = rados_object.get_pg_acting_set(pool_name=pool_name)
+        msg_acting_set = f"The {pool_name} pool acting set is -{acting_pg_set}"
+        log.info(msg_acting_set)
+
+        # Check that the scrubbing is progress or not
+        scrub_object.wait_for_pg_scrub_state(pg_id, wait_time=wait_time)
+
+        default_threshold_value = mon_obj.get_config(
+            section="osd", param="osd_scrub_load_threshold"
+        )
+
+        if float(default_threshold_value) != 10.0:
+            log.error(
+                "The default value of the osd_scrub_load_threshold is not equal to 10.0"
+            )
+            return 1
+        log.info("The default value of the osd_scrub_load_threshold is 10.0")
+
+        log.info(
+            "===== Scenario 1: Verification of the osd_scrub_load_threshold feature with the value 0"
+        )
+        log.info(
+            "Scenario 1: Setting the osd_scrub_load_threshold parameter value to 0"
+        )
+        # Perform the tests by modifying the
+        mon_obj.set_config(section="osd", name="osd_scrub_load_threshold", value=0)
+
+        # Truncate the osd logs
+        rados_object.remove_log_file_content(osd_nodes, daemon_type="osd")
+        log.info("Scenario1: The content of osd log are removed")
+        # set the debug log to 20
+        mon_obj.set_config(section="osd", name="debug_osd", value="20/20")
```

Comment:
update the debug_osd on only the OSDs that are part of the PG, collected from `acting_pg_set`. this will reduce the overall size of the logs on the cluster.